src/test/java/org/mdbenefits/app/data/SubmissionTestBuilder.java

```diff
@@ -1,12 +1,12 @@
 package org.mdbenefits.app.data;
 
 import formflow.library.data.Submission;
-import formflow.library.inputs.FieldNameMarkers;
+import org.mdbenefits.app.data.enums.CitizenshipStatus;
```

Comment:
This seems to have accidentally snuck in,

Reply:
Turned off my "Optimize imports on the fly setting", but it might be good if we used a standardized code format instead so that it just avoids conflicts.